### ISSUE TITLE
'rbw list: failed to decrypt' error, even though `rbw list` works

### ISSUE THREAD

**Issue by augustebaum** [fdw/rofi-rbw#99]
Running `rofi-rbw` fails, with the following messages:
```
There was a problem calling rbw. Is it correctly configured?
WARN: failed to decrypt username: failed to decrypt: failed to decrypt encrypted secret: invalid mac
WARN: failed to decrypt password: failed to decrypt: failed to decrypt encrypted secret: invalid mac
rbw list: failed to decrypt: failed to decrypt encrypted secret: invalid mac
```

```sh
$ rbw --version
rbw 1.12.1
```

```sh
$ rofi-rbw --version
rbw 1.4.2
```

`rbw list` works properly.

`rbw config show` shows:
```json
{
  "email": "<my-email>",
  "sso_id": null,
  "base_url": null,
  "identity_url": null,
  "ui_url": null,
  "notifications_url": null,
  "lock_timeout": 3600,
  "sync_interval": 3600,
  "pinentry": "pinentry",
  "client_cert_path": null
}
```

**Comment by fdw** [comment]
That error only pops up when `rbw` doesn't exit cleanly. Can you please try `rbw list --fields folder,name,user`? This is what `rofi-rbw` calls, and your error also suggests that the problem is with username and password.

**Comment by augustebaum** [comment]
Thanks for the quick response!

I found and resolved my issue; rofi-rbw itself had nothing to do with it. I'll write it up here in case anyone ends up in the same situation.

TL;DR: make sure that the rbw called by rofi-rbw is the one you think (I used `strace rofi-rbw` to find that it wasn't).

I'm on NixOS, and I've installed both rofi-rbw and rbw on my machine separately; this way if I want to use rbw on its own it's on my PATH.
I overrode the versions of rbw and rofi-rbw to be the most recent ones, using in my Nix confix; this way, I can use the latest versions without having to upgrade my whole system at once, which can easily break things.
However, rofi-rbw was overriden in a different way than rbw, so my 1.4.2 rofi-rbw still used an older version of rbw (1.9.0), even though the rbw in my PATH was 1.12.1.
The solution was to put both overrides in `nixpkgs.config.packageOverrides`, so that the rbw in `rofi-rbw` is the same as the one in my PATH.
